src/custom/utils/explorer.ts

```diff
@@ -1,13 +1,31 @@
 import { ChainId } from '@uniswap/sdk'
 import { OrderID } from 'utils/operator'
+import { checkEnvironment } from './environments'
 
-// TODO: using dev endpoint for now. Update to staging/prod when available
-const EXPLORER_BASE_URL: Partial<Record<ChainId, string>> = {
-  [ChainId.MAINNET]: 'https://protocol-explorer.dev.gnosisdev.com',
-  [ChainId.RINKEBY]: 'https://protocol-explorer.dev.gnosisdev.com/rinkeby',
-  [ChainId.XDAI]: 'https://protocol-explorer.dev.gnosisdev.com/xdai'
+const { isDev, isStaging, isProd } = checkEnvironment(window.location.host)
+
+function _getExplorerUrlByEnvironment() {
+  let baseUrl: string | undefined
+  if (isDev) {
+    baseUrl = process.env.REACT_APP_EXPLORER_URL_DEV || 'https://protocol-explorer.dev.gnosisdev.com'
+  } else if (isStaging) {
+    baseUrl = process.env.REACT_APP_EXPLORER_URL_STAGING || 'https://protocol-explorer.staging.gnosisdev.com'
+  } else if (isProd) {
+    baseUrl = process.env.REACT_APP_EXPLORER_URL_PROD || 'https://gnosis-protocol.io'
+  } else {
+    // default to dev
```

Comment:
I don't believe this should be an error or anything app blocking. Also helpful in localhost